debian/sml_server.1.md

```diff
@@ -0,0 +1,54 @@
+% SML_SERVER(1) Volkszaehler project | User Commands
+% Joachim Zobel <jz-2017@heute-mogen.de>
+
+# NAME
+
+sml_server - makes libsml functionality available on the command line
+
+# SYNOPSIS
+
+**sml_server** [**-v**] [**-h**] [**-s**] _device_|_file_
+
+# DESCRIPTION
+
+The sml_server reads binary SML data from the given device or file. If a - is 
+given, stdin is used. This data is then displayed in human readable form.
+
+**-v**
+:     This enables verbose output.
+
+**-h**
+:     This shows the help.
+
+**-s**
+:     will print the first sml message and then exit (single)
+
+# EXAMPLES
+
+~~~
+$ sml_server /dev/ttyUSB0 
```

Comment:
i think this might be more confusing than useful,
it this should at least mention that it requires a meter connected to that port that is actively sending data.
if you go with the rationale of providing a working default config, you could ship an example sml dump with the package ;)